static/app/stores/repositoryStore.tsx

```diff
@@ -3,32 +3,28 @@ import Reflux from 'reflux';
 import RepoActions from 'app/actions/repositoryActions';
 import {Repository} from 'app/types';
 
-type RepositoryStoreInterface = {
-  get(): {
-    orgSlug?: string;
-    repositories?: Repository[];
-    repositoriesLoading?: boolean;
-    repositoriesError?: Error;
-  };
-
-  state: {
-    orgSlug?: string;
-    repositories?: Repository[];
-    repositoriesLoading?: boolean;
-    repositoriesError?: Error;
-  };
+type State = {
+  orgSlug?: string;
+  repositories: Repository[];
+  repositoriesLoading: boolean;
+  repositoriesError?: Error;
+};
 
+type RepositoryStoreInterface = {
```

Comment:
you can type this as `type RepositoryStoreInterface = CommonStoreInterface<State> & {...}` and remove the getState() type def here to match the other stores